src/app/api/sse/test/route.ts

```
@@ -0,0 +1,289 @@
+          `id: ${crypto.randomUUID()}`,
+          `event: system_notification`,
+          `data: ${JSON.stringify({
+            title: "Test Notification",
+            message:
+              "This is a test notification sent via the SSE test interface",
+            type: "info",
+            timestamp: new Date().toISOString(),
+          })}`,
+          "",
+          "",
+        ].join("\n");
```

Biome (2.4.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)